holmes.go

```diff
@@ -73,11 +73,13 @@ func main() {
 	//only prepend the protocol when the user provided one
 	if *protocol != "" {
 		baseURL = *protocol + "://"
+	} else {
+		baseURL = "//"
```

Comment:
Unfortunately, you can't just prepend "//" since it must be possible to leave the baseURL empty for #12